specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -0,0 +1,1524 @@
+          "maxLength": 128,
+          "minLength": 0,
+          "type": "string"
+        },
+        "includeSubFolders": {
+          "description": "A flag to indicate if sub folders within the set of\r\nprefix folders will also need to be included when searching\r\nfor content to be preprocessed.",
+          "type": "boolean",
+          "default": false,
+          "x-nullable": false
+        }
+      }
+    },
+    "TrainResult": {
+      "description": "Response of the Train API call.",
+      "type": "object",
+      "required": [
+        "trainingDocuments",
+        "errors"
+      ],
+      "properties": {
+        "trainingDocuments": {
+          "description": "List of documents used to train the model and the\r\ntrain operation error reported by each.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormDocumentReport"
+          }
+        },
+        "trainingFields": {
+          "description": "Training Fields.",
+          "$ref": "#/definitions/TrainingFields"
+        },
+        "errors": {
+          "description": "Errors returned during the training operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormOperationError"
+          }
+        }
+      }
+    },
+    "TrainingFields": {
+      "description": "Training Fields.",
+      "type": "object",
+      "required": [
+        "fields",
+        "averageModelAccuracy"
+      ],
+      "properties": {
+        "fields": {
+          "description": "List of fields used to train the model and the\r\ntrain operation error reported by each.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormFieldsReport"
+          }
+        },
+        "averageModelAccuracy": {
+          "description": "Average accuracy.",
+          "type": "number"
+        }
+      }
+    },
+    "SourcePath": {
+      "description": "URI or local path to source data.",
+      "type": "object",
+      "properties": {
+        "source": {
+          "description": "Get or set source path to file to score.",
+          "maxLength": 2048,
+          "minLength": 0,
+          "type": "string"
+        }
+      }
+    },
+    "SourceDataStream": {
+      "description": "A pdf document or image (jpg,png,tiff) file to analyze.",
+      "type":"object"
+    },
+    "ModelInfo":{
+      "description": "A pdf document or image (jpg,png,tiff) file to analyze.",
+      "type":"object",
+      "required": [
+        "modelId",
+        "status",
+        "createdDateTime",
+        "lastUpdatedDateTime"
+      ],
+      "properties": {      
+        "modelId": {
+          "description": "Get or set model identifier.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "status": {
+          "description": "Get or set the status of model.",
+          "enum": [
+            "creating",
```

Comment:
Do we need a status "not started"?

Reply:
Hmm.. I thought about that but could not see it adding much value. In cases where /Train requests being not-started for few minutes could lead to customers to call support.

Reply:
So if the task is in queue, we return creating? kind of strange.